src/rapala.py

```diff
@@ -103,24 +92,23 @@ def __on_article_action(self) -> None:
         """
         This func contains all actions that happen when an article is opened
         """
-        # driver.wait.until(EC.visibility_of_element_located((By.TAG_NAME, 'p')))
-        # grab & parse page html
+        
         content = self.driver.page_source
         soup = bs(content)
-        # included to write article title and category into file
+       
         title = None
 
         if title:
```

Comment:
You can do without initializing `title` to None. Simply attempt to retrieve it as you did in 102 - 104. Then check if it is None before falling back to what you did in 107.

Reply:
fixed this too, thanks for pointing that out.